jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -36,17 +36,8 @@ If you need more organizations or projects, consider upgrading to a xref:plan-ov
 
 Follow the steps on the xref:first-steps#[Sign up and try CircleCI] page to connect your GitHub account, which creates your CircleCI organization.
 
-When you create a new organization and connect your GitHub account, you will be prompted to create a new project from a repository. You will be presented with the following options to set up a CircleCI configuration for your project:
+When you create a new organization and connect your GitHub account, you will be prompted to create a new project from a repository. After creating a new project, CircleCI will automatically generate a custom configuration file based on the programming languages and frameworks detected in your repository. You will be presented with the following options to either commit and run the generated configuration file on a new branch or view the generated configuration file in the configuration editor. 
```

Comment:
The "You will be presented with the following options" part .... what options will they see?  With the removal of the "faster" and "fast" options I don't think there is anything listed right now. If you have a recording or something of what happens at this point I can help iron this out?

Reply:
Ah, this step is referencing after users click the 'Create Project' CTA. So we will be redirecting them to a new loading screen page while we're generating their custom config file. I'll DM you via Slack with more details of the flow!